analyses/cms-open-data-ttbar/analysis.py

```diff
@@ -348,7 +348,7 @@ def main() -> None:
         client = None
         load_cpp()
         if args.inference:
-            ml.load_cpp("./fastforest")
+            ml.load_cpp()
```

Comment:
```suggestion
```
You don't need this if statement at all now

Reply:
I still need this function to get some other ml helpers though

Reply:
Oh, I see